packages/thefuck.rb

```diff
@@ -0,0 +1,14 @@
+require 'package'
+
+class thefuck < Package
+  description 'Magnificent app which corrects your previous console command.'
+  homepage 'https://github.com/nvbn/thefuck'
+  version '3.26'
+
+  depends_on 'python3'
+
+  def self.install
+    system "python3 -m pip install thefuck"
```

Comment:
We'll need the `--root` and `--prefix` options also, otherwise, the package cannot be uninstalled via crew.
```
system "python3 -m pip install thefuck --root=#{CREW_DEST_DIR} --prefix=#{CREW_PREFIX}"
```

Reply:
fixed